eZ/Publish/API/Repository/Values/Content/Trash/SearchResult.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace eZ\Publish\API\Repository\Values\Content\Trash;
+
+use ArrayIterator;
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+class SearchResult extends ValueObject implements \IteratorAggregate
+{
+    public function __construct(array $properties = [])
+    {
+        if (isset($properties['totalCount'])) {
+            $this->count = $properties['totalCount'];
+        }
+
+        parent::__construct($properties);
+    }
+
+    /**
+     * The total number of Trash items.
+     *
+     * @var int
+     */
+    public $totalCount = 0;
```

Comment:
Self-note: We also should adapt https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/lib/Pagination/Pagerfanta/TrashItemAdapter.php#L17 to this change.

Reply:
Yes, that adapter is the reason I kept count in here for now, but also for other places that might use it :)